src/System.Numerics.Vectors/tests/Vector2Tests.cs

```diff
@@ -0,0 +1,1199 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Numerics.Tests
+{
+    public class Vector2Tests
+    {
+        [Fact]
+        public void Vector2MarshalSizeTest()
+        {
+            Assert.Equal(8, Marshal.SizeOf<Vector2>());
+            Assert.Equal(8, Marshal.SizeOf<Vector2>(new Vector2()));
+        }
+
+        [Fact]
+        public void Vector2CopyToTest()
+        {
+            Vector2 v1 = new Vector2(2.0f, 3.0f);
+
+            float[] a = new float[3];
+            float[] b = new float[2];
+
+            Assert.Throws<NullReferenceException>(() => v1.CopyTo(null, 0));
+            Assert.Throws<ArgumentOutOfRangeException>(() => v1.CopyTo(a, -1));
+            Assert.Throws<ArgumentOutOfRangeException>(() => v1.CopyTo(a, a.Length));
+            
+            if (!PlatformDetection.IsNetNative)
```

Comment:
This PlatformDetection property does not currently exist. Should I add it? The value is:

```cs
        public static bool IsNetNative => RuntimeInformation.FrameworkDescription.StartsWith(".NET Native", StringComparison.OrdinalIgnoreCase);
```

But I am unsure if we need it.

For now, I'm going to remove it, as it is affecting the ability to build.

Reply:
Depends on what's there and how much of the test codebase needs it.  We're getting close to the end now, so maybe folks didn't need it?

Reply:
Well, I only found the one use. And haven't seen it in any of my other PRs (yet).

Reply:
Oh, I see - it's only a single property.  Delete the code :)

Reply:
Deleted